src/commands/flags.mjs

```diff
@@ -367,7 +367,7 @@ export const keyFormat = {
   constName: 'keyFormat',
   name: 'key-format',
   definition: {
-    describe: 'Public and Private key file format (pem or pfx)',
```

Comment:
it seems that keyFormat is no longer needed since there is only one option, can it be removed from everywhere, also?

Reply:
Removed